STYLEGUIDE.md

```diff
@@ -888,6 +883,38 @@ When a node has multiple attributes that would cause it to exceed the line chara
 </ul>
 ```
 
+# Api Style Guide
+
+## Paths
+
+API routes must start with the `/api/` path segment, and should be followed by the plugin id if applicable:
+
+*Right:* `/api/kibana/index_patterns`
+*Wrong:* `/api/kibana_index_patterns`
+
+## Versions
+
+Kibana won't be supporting multiple API versions, so API's should not define a version.
+
+*Right:* `/api/kibana/index_patterns`
+*Wrong:* `/marvel/v1/api`
```

Comment:
This right and wrong doesn't look like an apples-to-apples comparison. We're just saying that Kibana APIs shouldn't define a version, so should the "wrong" look like: `/api/kibana/v1/index_patterns`?